truss/templates/control/control/endpoints.py

```diff
@@ -94,6 +92,59 @@ async def proxy(request: Request):
     return response
 
 
+def inference_retries(
+    retry_condition: Callable[[RetryCallState], bool] = BASE_RETRY_EXCEPTIONS,
```

Comment:
Wanted this to be extensible in the future, even though both ws/http use the same retry logic today